Assets/MixedRealityToolkit.Examples/Experimental/HandMenuLayout/Prefabs/HandMenuSliderAssets/Scripts/UpdateTrackScale.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
```

Comment:
Missing license header

Reply:
Here is the header to add:

// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See LICENSE in the project root for license information.

Thanks!